pysite/tables.py

```diff
@@ -0,0 +1,19 @@
+from typing import List, NamedTuple
+
+
+class Table(NamedTuple):
+    primary_key: str
+    keys: List[str]
+    locked: bool = True
+
+
+TABLES = {
+    "hiphopify": Table("user_id", sorted(["user_id", "end_timestamp", "forced_nick"])),
```

Comment:
I don't like these. I think they're too dense, and I don't understand why you're using sorted instead of just listing them alphabetically. I think I'd prefer these multilined with a blank line seperating them or something like that.

How about something like

```py
"hiphopify": Table(
    primary_key="user_id", 
    keys=[
        "end_timestamp", 
        "forced_nick",
        "user_id",
    ]
),

"hiphopify_namelist": Table(
    primary_key="name", 
    keys=[
        "image_url",
        "name",
    ], 
    locked=False
),
```